apps/src/dance/ai/dance-ai-modal.module.scss

```diff
@@ -214,15 +236,29 @@ $ai-modal-border-width: 8px;
         .botHead {
           position: absolute;
           width: 97px;
-          left: 81.5px;
           top: 34px;
-          transform-origin: bottom left;
           transition: transform 0.5s 0.25s;
           transform: rotate(0deg);
           pointer-events: none;
 
+          html[dir='ltr'] & {
+            left: 81.5px;
```

Comment:
super nit: but why the 0.5 px difference between ltr and rtl?

Reply:
Great question.  Because of the original art assets we received (the head image not as wide as the body image), we had to bump the head by a subpixel to make the it line up perfectly with the body in LTR.  But in RTL, it lined up perfectly with a slightly different value.  In hindsight, I should have requested slightly different assets that lined up more easily.